### PR TITLE
Adds more clarity around the current user object on ftl theme pages

### PR DESCRIPTION
**Pull request by jobannon**

### CODE REVIEW

File: site/docs/v1/tech/themes/template-variables.adoc [hunk 1]
```diff
@@ -36,7 +36,7 @@ The application resolved by the provided [field]#client_id# provided on the requ
 The OAuth v2.0 `client_id` parameter. This is synonymous with FusionAuth's Application Id.
 
 [field]#currentUser# [type]#[User]# [since]#Available since 1.30.0#::
-When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be undefined.
+When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be undefined.  In other words, this variable availability is synonymous with the `Keep me signed in` login option.
```
Comment (mooreds) [inline site/docs/v1/tech/themes/template-variables.adoc:39]:
```suggestion
When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be null. If the user has not checked the `Keep me signed in` option, there is no SSO session and this variable will be null.
```

**Review by mooreds — APPROVED** [review]
Wording change suggestion, otherwise looks good to me.

File: site/docs/v1/tech/themes/template-variables.adoc [hunk 2]
```diff
@@ -36,7 +36,7 @@ The application resolved by the provided [field]#client_id# provided on the requ
 The OAuth v2.0 `client_id` parameter. This is synonymous with FusionAuth's Application Id.
 
 [field]#currentUser# [type]#[User]# [since]#Available since 1.30.0#::
-When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be undefined.
+When there is an active SSO session, this variable will contain the currently logged in user. When an SSO session does not yet exist, this variable will be null. If the user has not checked the `Keep me signed in` option, there is no SSO session and this variable will be null.
```
Comment (robotdan) [inline site/docs/v1/tech/themes/template-variables.adoc:39]: null should be `null`.

Comment (jobannon) [reply site/docs/v1/tech/themes/template-variables.adoc:39]: Updated and merged in a separate PR. Thanks!

**Pull request merged**